doc/api/util.md

```diff
@@ -1053,6 +1056,7 @@ changes:
       times. If `true`, all values will be collected in an array. If
       `false`, values for the option are last-wins. **Default:** `false`.
     * `short` {string} A single character alias for the option.
+    * `required` {boolean} Whether this option is required. **Default:** `false`.
```

Comment:
Doesn't the default depend on `strict`?

Reply:
I’m not sure what it should be🤔
But in my real code, I just check if it is a truely value.

Reply:
No, I think the behaviour is independent of `strict`. Options which take a value generate an error in `strict`, but there is a difference between an option with a required option-argument (i.e. `type='string'`) and a required option as proposed here.